src/encoded/static/components/dbxref.js

```diff
@@ -65,9 +65,9 @@ export const dbxrefPrefixMap = {
     HGNC: {
         pattern: 'http://www.genecards.org/cgi-bin/carddisp.pl?gene={0}',
         preprocessor: (context) => {
-            // For dbxrefs in targets, use target.gene_name instead of the dbxref value.
-            if (context['@type'][0] === 'Target' && context.gene_name) {
-                return { altValue: context.gene_name };
+            // For dbxrefs in targets, use the symbol of the first gene in target.genes instead of the dbxref value.
+            if (context['@type'][0] === 'Target' && context.genes) {
+                return { altValue: context.genes[0].symbol };
```

Comment:
If target.genes exists, are you guaranteed to have a first item (context.genes[0]) or could there be a case that this array is empty?

Reply:
Initially, I think this is OK because target.genes shouldn't be an empty array if the schema and dependency works properly. However, now I think you are absolutely right. I should never rely on that in the code. I'll add ` && context.genes.length > 0`.

Reply:
One more clarification for this question. If target.genes is indeed an empty array, there is a chance the link goes bad (because there is no good altValue). I think this is expected and (has to be) acceptable.